maprendering/maprendering.dm

```diff
@@ -0,0 +1,109 @@
+/client/proc/maprender()
+	set category = "Mapping"
+	set name = "Generate Map Render"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(alert("Sure you want to do this? It should NEVER be done in an active round and cannot be cancelled", "generate maps", "Yes", "No") == "No")
```

Comment:
Yep pretty sure he did do nano map gen during a round before.

Let's not forget him to trying to move every turf with SDQL. 

Celt shouldn't have +DEBUG